src/mayara_pi.cpp

```
@@ -384,26 +384,15 @@ void mayara_pi::LoadConfig() {
-  cfg->Read("FixedHeading", &fh, 0.0);
   cfg->Read("CogAsHeading", &cog, false);
-  cfg->Read("FixedPosition", &fpos, false);
-  cfg->Read("FixedLat", &fla, 0.0);
-  cfg->Read("FixedLon", &flo, 0.0);
   cfg->Read("LogLevel", &lvl, 0);
-  m_diag.heading_source =
-      static_cast<int>(hs < 0 ? 0 : (hs > Diagnostics::kFixedHeading
-                                         ? Diagnostics::kFixedHeading
-                                         : hs));
+  m_diag.heading_source = static_cast<int>(
+      hs < 0 ? 0 : (hs > Diagnostics::kRadarOnly ? Diagnostics::kRadarOnly : hs));
```

Clang (14.0.6)

[note] +1, including nesting penalty of 0, nesting level increased to 1

(clang)